### PR TITLE
Basics hometask done.

### PR DESCRIPTION
**Pull request by kusem**

### CODE REVIEW

File: src/basics/basics.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getMinuteQuarter(int $minute): string
+    {
+        $this->validator->isMinutesException($minute);
+
+        if ($minute > 45 | $minute == 0) {
```
Comment (yderevianko-cartera) [inline src/basics/basics.php]: should be  || instead of |

File: src/basics/basics.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            return "fourth";
```
Comment (yderevianko-cartera) [inline src/basics/basics.php]: please use single quote for strings 'fourth'

File: src/basics/basics.php [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        $firstHalf =
+            (int)$convertedString[0] +
+            (int)$convertedString[1] +
+            (int)$convertedString[2];
+        $secondHalf =
+            (int)$convertedString[3] +
+            (int)$convertedString[4] +
+            (int)$convertedString[5];
+
+        return $firstHalf == $secondHalf;
```
Comment (yderevianko-cartera) [inline src/basics/basics.php]: please use strict comparison

**Review by yderevianko-cartera — CHANGES_REQUESTED** [review]

File: src/basics/basics.php (hunk 3 above)
Comment (kusem) [reply src/basics/basics.php]: Done, next commit

File: src/basics/basics.php (hunk 2 above)
Comment (kusem) [reply src/basics/basics.php]: > please use single quote for strings 'fourth'

Done, next commit

File: src/basics/basics.php (hunk 1 above)
Comment (kusem) [reply src/basics/basics.php]: > should be || instead of |

Done, next commit

File: src/basics/BasicsValidator.php [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+use InvalidArgumentException;
+
+class BasicsValidator implements BasicsValidatorInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function isMinutesException(int $minute): void
+    {
+        if ($minute < 0 or $minute >= 60) {
```
Comment (yderevianko-cartera) [inline src/basics/BasicsValidator.php:14]: never use or, please change it to ||

**Review by yderevianko-cartera — APPROVED** [review]

**Pull request merged**